lib/Onyx.ts

```diff
@@ -642,7 +678,10 @@ function update(data: OnyxUpdate[]): Promise<void> {
         }
     });
 
-    return clearPromise.then(() => Promise.all(promises.map((p) => p()))).then(() => undefined);
+    return clearPromise
+        .then(() => Promise.all(promises.map((p) => p())))
+        .then(() => updateSnapshots(data))
```

Comment:
Yea, I think that's fine since we only want snapshot data to be updated from Pusher updates, which calls `Onyx.update`. Are there other edge cases where the API calls `merge` instead of `update`?

Reply:
I don't see any edge case right now but I think it can be confusing if one forgets about this fact. After all it's open-source library